worktrolly-ui/src/app/app.component.html

```diff
@@ -2,4 +2,4 @@
       rel="stylesheet">
 <app-header></app-header>
 <app-body></app-body>
-<router-outlet></router-outlet>
+<router-outlet></router-outlet>
```

Comment:
Please revert this file as well

Reply:
But this file has the google material icon cdn link